packages/canon/src/components/box/box.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createElement } from 'react';
+import { boxSprinkles } from './sprinkles.css';
+import { base } from './box.css';
+
+/**
+ * Properties for {@link Box}
+ *
+ * @public
+ */
+export type BoxProps = Parameters<typeof boxSprinkles>[0] &
+  Omit<
+    React.AllHTMLAttributes<HTMLElement>,
+    keyof Parameters<typeof boxSprinkles>[0]
+  > & {
+    as?: keyof JSX.IntrinsicElements;
+  };
+
+/** @public */
+export const Box = ({ as = 'div', className, style, ...props }: BoxProps) => {
```

Comment:
Please deconstruct arguments within the function instead, so that this is just `props: BoxProps`. It makes API reports and docs neater

Reply:
That's done.